### ISSUE TITLE
Hybrid Deployment Agent in Kubernetes

### ISSUE THREAD

**Issue by dscott-lh** [fivetran/hybrid_deployment#4]
Is there support for running the hybrid deployment agent in Kubernetes?

With the exception of mounting the docker/podman socket into the container, everything I see in install.sh and hdagent.sh can be replicated in a Kubernetes pod.

Is the hybrid deployment agent using the docker/podman socket to run additional containers?
If so, is there any plan to implement a native Kubernetes controller?

**Comment by fivetran-antonels** [comment]
Yes Kubernetes support is planned and should be available soon.   The Agent will be deployed using helm chart and will create the required jobs as Kubernetes jobs. 
More detail will be added to the repository within next few weeks with all the details.
